.stylelintrc

```diff
@@ -16,6 +15,7 @@
         "ignoreAtRules": ["else"]
       }
     ],
+    "declaration-block-trailing-semicolon": null,
```

Comment:
This started acting up -- it was incorrectly thinking I should add a comma after things like `style={{ display: 'block' }}`. I'm turning it off